packages/editor/src/lib/editor/managers/LicenseManager.ts

```diff
@@ -14,23 +26,31 @@ export type InvalidLicenseReason = 'invalid-license-key' | 'no-key-provided'
 export type LicenseFromKeyResult = InvalidLicenseKeyResult | ValidLicenseKeyResult
 
 interface InvalidLicenseKeyResult {
-	isLicenseValid: false
+	isLicenseParseable: false
 	reason: InvalidLicenseReason
 }
 
 interface ValidLicenseKeyResult {
-	isLicenseValid: true
+	isLicenseParseable: true
 	license: LicenseInfo
+	isDevelopmentKey: boolean
 	isDomainValid: boolean
+	isAnnualLicense: boolean
+	isPerpetualLicense: boolean
+	isInternalOnly: boolean
 	isLicenseExpired: boolean
 }
 
 export class LicenseManager {
-	private publicKey = '3UylteUjvvOL4nKfN8KfjnTbSm6ayj23QihX9TsWPIM='
+	private publicKey =
+		'-----BEGIN PUBLIC KEY-----\nMHYwEAYHKoZIzj0CAQYFK4EEACIDYgAEPkmEjocP8ldvaSv6BZuhhl+KgrBPn15eckpnYTtVGyqUngQnqdca/4BdZuCwxBR84cvE0MDQ/VnOu/Fyh+K2xr/uewxKqp9OaqqsGnedNdi4ypMZEnWIZkH32wn5BP6W\n-----END PUBLIC KEY-----'
```

Comment:
Do we need the prefix and suffix here? I'd just use the key so we don't have to remove prefix/sufix later. I guess it's harder to mess up pasting a new key if it came to that?

Reply:
it's not a bad point. i started removing it just now but it's tricky b/c then the keys can start to get mixed up. i think keeping it consistent with the PEM standard will result in less footguns later on even if it's a bit more verbose. 🤞